gcc/rust/ast/rust-ast-dump.cc

```diff
@@ -554,7 +579,23 @@ Dump::visit (Method &method)
 
 void
 Dump::visit (Module &module)
-{}
+{
+  indentation.increment ();
```

Comment:
I feel this should be after the curly brace (line 588), to indent the body only.

Reply:
Also a space after mod is missing.

Reply:
Fixed in [6bb6550](https://github.com/Rust-GCC/gccrs/pull/1548/commits/6bb6550c44d6644bb22e4a66be467802faeb23d8)